harbor-registry.yaml

```diff
@@ -57,7 +67,8 @@ test:
         test $(curl -LI localhost:5000 -o /dev/null -w '%{http_code}\n' -s) == "200"
 
 update:
-  enabled: true
+  # Re-enable when v3.x.x lands a stable release
+  enabled: false
```

Comment:
If you change this to `manual: true` it'll open an issue at least, to remind folks to take a look.